src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,11 @@
+$links-color: #00ACDC;
+$mail-link-color: #060B35;
+$card-size: 200px;
+$button-bg-color: #00acdc;
+$button-color: #fff;
+$button-border-color: #00acdc;
+$card-border-color: #f3f3f3;
+$button-hover-color: #00acdc;
+$text-main-color: #060b35;
```

Comment:
still not fixed, you can create one variable($primary-color: #00acdc) and use it in different places instead of creating few variables with same values for each style